### ISSUE TITLE
add a UI interface, similar to refined storage.

### ISSUE THREAD

**Issue by butrdukr** [jaquadro/StorageDrawers#1138]
pretty please

**Comment by Jack-McKalling** [comment]
No, this is just a storage mod. If you really like to have a rich interface to browse through all the contents of the network, like Refined Storage, you can hook up the mod to offer said terminal.

Applied Energistics, Refined Storage, Simple Storage Network, Integrated Terminals, all support Storage Drawers as a "storage network" that can be interacted with using a storage bus, link cable, etc. Why exactly do you want SD to introduce its own concurrent implementation?
